lib/concepts/class_member/operations/list.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class ClassMember
+module ClassMember
```

Comment:
I'm being curios, why do switch class here to a module?

Reply:
Because we renamed the class_member table in the database to class_student, and also renamed the ClassMember model/class to ClassStudent. So we don't really care about creating instances of ClassMember anymore, just in using the methods it provides. These methods in some cases were taking account of both teachers and students, even though instances of the old ClassMember model could only represent students, which was confusing, hence the change.

Reply:
Thanks for open anwer👏